### ISSUE TITLE
Upload best model for prediction?

### ISSUE THREAD

**Issue by JacobLenzing** [ohuelab/SpatialPPI#3]
Is it possible for you to upload the "best_model.hdf5" referenced in the code?

Thank you in advance.

**Comment by anne04** [comment]
Yes, I have the same question! Could you please upload the "best_model.hdf5" referenced in the code?
